src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Math/BigIntStruct.fs

```diff
@@ -1,374 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This file was not included in any active testing. BigInteger is now part of the .NET Framework and tested there.  Also we have some tests in BigIntType.fs